pkg/process/metadata/parser/service_windows_test.go

```diff
@@ -58,14 +58,15 @@ func TestWindowsExtractServiceMetadata(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			mockConfig := ddconfig.MockSystemProbe(t)
 			mockConfig.SetWithoutSource("system_probe_config.process_service_inference.enabled", true)
-
+			enabled := mockConfig.GetBool("system_probe_config.process_service_inference.enabled")
+			useWindowsServiceName := mockConfig.GetBool("system_probe_config.process_service_inference.use_windows_service_name")
```

Comment:
Please also remove the mock config from this test as it's not necessary

Reply:
updated!